contrib/cirrus/pr-should-link-jira

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+#
+# Intended for use in CI, for -rhel branches: check git PR, barf if no jira links
+#
+
+ME=$(basename $0)
+
+set -e
+
+# Github label which allows overriding this check
+OVERRIDE_LABEL="No Jira Link"
+
+# Only -rhel branches need jira links
+BRANCH_REGEX="v.*-rhel"
```

Comment:
Only Podman has the `-rhel` branch thought.  If you want to do other branches, this will have to be adjusted somehow.

Reply:
Not sure what kind of branches you mean, care to give an example?